tools/mktutorial/main.go

```diff
@@ -271,6 +271,9 @@ func makeLangMap(tutorials []tutorial, include []string, exclude []string) (map[
 		"js": nil,
 		"ts": nil,
 		"py": nil,
+		"go": nil,
+		"cs": nil,
+		"fs": nil,
```

Comment:
I don't see what I would expect structurally in the rest of this PR to support F#? For example, `cloud_index.tmpl`  only has a C# block and similarly, we don't include "fs" in our map for rendering the templates. If you're intentionally leaving it out, I don't think there's any need to account for it in this map.

Reply:
I can remove the `fs`.  Included here just so that it's easier to add support in the future if we want (this list has all the languages we have examples for - the other lists use a subset of those to present in UI).  But can leave this out.